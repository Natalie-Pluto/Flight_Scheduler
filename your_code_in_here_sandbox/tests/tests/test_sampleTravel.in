location import locations.csv
location import locations4.csv
flight import flights6.csv
travel sydney london
travel mars london
travel london moon
travel
location add shanghai 31.23 121.47 0.5
location add Hongkong 22.32 114.17 0.5
travel shanghai Hongkong
exit