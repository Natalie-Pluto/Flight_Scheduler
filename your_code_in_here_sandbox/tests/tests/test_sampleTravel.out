User: Imported 51 locations.

User: Imported 2 locations.

User: Imported 182 flights.

User: Legs:             3
Total Duration:   34h 36m
Total Cost:       $5172.49
-------------------------------------------------------------
ID   Cost      Departure   Arrival     Source --> Destination
-------------------------------------------------------------
   5 $ 3399.00 Wed 10:00   Thu 02:44   Sydney --> AbuDhabi
LAYOVER 1h 16m at AbuDhabi
  10 $ 1384.44 Thu 04:00   Thu 11:11   AbuDhabi --> Oslo
LAYOVER 7h 49m at Oslo
  20 $  389.05 Thu 19:00   Thu 20:36   Oslo --> London

User: Starting location not found.

User: Ending location not found.

User: Usage: TRAVEL <from> <to> [cost/duration/stopovers/layover/flight_time]

User: Successfully added location shanghai.

User: Successfully added location Hongkong.

User: Sorry, no flights with 3 or less stopovers are available from shanghai to Hongkong.

User: Application closed.
